iot-hub/Samples/service/ImportExportDevicesSample/ImportExportDevicesSample.cs

```diff
@@ -18,31 +18,30 @@ namespace Microsoft.Azure.Devices.Samples
 {
     public class ImportExportDevicesSample
     {
-        // Container used to hold the blob containing the list of import/export files.
-        // This is a module-wide variable. If this project doesn't find this container, it will create it.
-        private CloudBlobContainer _cloudBlobContainer;
-
-        // Name of blob container holding the work data.
-        private const string ContainerName = "devicefiles";
-
-        // Name of the file used for exports and imports. 
-        // This is set by the IoT SDK, and can't be changed.
-        private const string DeviceListFile = "devices.txt";
-
-        private string _containerUriWithSas = string.Empty;
-
         private readonly string _iotHubConnectionString;
```

Comment:
Can we change this as well?

```suggestion
        private readonly string _srcIotHubConnectionString;
```